### ISSUE TITLE
Sharpe/Sortino period needs to be 252 for financial markets

### ISSUE THREAD

**Issue by Koha101** [Lumiwealth/quantstats_lumi#30]
The period has been changed from 252 in quantstats to 365 in quanstats_lumi, to correct the CAGR calculation, since it uses calendar days in its calculation. This breaks the sharpe and sortino calculations, since they use the sqrt(period), which should be the number of trading days, not calendar days, 252 for financial markets. The 365 number works for crypto, since that's available 365 days/year.

I propose the default should be 252 for sharpe and sortino, and 365 for CAGR. Additionally, it would be worth having 2 separate periods in reports.py, calendar_period and trading_period, to use the correct one on calls to the metrics, and expose the two periods in calls like reports.html().

**Comment by jakewillms17** [comment]
I disagree with using 365 and 252 like this. Very simple fix for CAGR would be to just count the len(returns) to gather the number of days as opposed to the calendar calculation. Makes more sense IMO to standardize 252 as we are dealing with financial data.

**Comment by jiema** [comment]
252 is correct imo, and 365 is wrong.
